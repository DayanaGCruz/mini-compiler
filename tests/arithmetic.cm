void main() {
  int a;
  int b;
  int c;

  a = 10;
  b = 3;
  c = a - b;
  print(c);

  c = a + b - 2;
  print(c);

  c = a - b - 2;
  print(c);

  c = 5 - (a - b);
  print(c);
}
