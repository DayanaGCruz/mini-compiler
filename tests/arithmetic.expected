7
11
5
-2
